.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,18 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+      - 'releases/**'
```

Comment:
Do we need the `releases` and `stable` branches? These don't currently exist in this repo.

Reply:
That is due to copy/paste but it would not hurt the project in any way. It would also document its purpose. We do not want generic `**` here because we do not want to accidentally make a release from a temporary/feature branch that we created using a push.